### PR TITLE
bump dependency-funnel

### PR DESCRIPTION
**Pull request by stefanpenner**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -74,7 +74,7 @@
     "broccoli-babel-transpiler": "^5.6.1",
     "broccoli-concat": "^3.0.1",
     "broccoli-debug": "^0.6.0",
-    "broccoli-dependency-funnel": "^1.0.2",
+    "broccoli-dependency-funnel": "2.0.0",
```
Comment (rwjblue) [inline package.json]: @stefanpenner - The only breakage here was RE: dropping Node 0.12 support, right?

Comment (rwjblue) [inline package.json]: Also, can we use `^` instead of exact match?  Just in case we need to fix any bugs or something...

**Comment by stefanpenner** [comment]
I dont understand this failure :(

**Comment by stefanpenner** [comment]
NM, think I figured it out.... `files` list in broccoli-dependency-funnel wasn't up2date.

**Comment by rwjblue** [comment]
Files trolls so hard 😩. I wish there was someway to prevent it from hitting so hard (perhaps an eslint plugin or something?)...

**Pull request merged**